### PR TITLE
Add Job Information to Job Page Metadata

### PR DESCRIPTION
**Pull request by jhancock532**
### Description of Changes Made

The job page was missing metadata titles and descriptions relevant to each job, this MR fixes this. 

### How to Test

[View a job listing on the preview site](https://careers-iapbehtmq-careers-site.vercel.app/careers/jobs/junior-developer) - you can also use tools such as [https://metatags.io/](https://metatags.io/) to preview how the page will appear on social media.

### Screenshots

<details>
  <summary>Expand to show more</summary>
image

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
- [x] Tests and linting passes.
- [x] Consider updating documentation. If you don't, tell us why.
- [x] If relevant, list the environments / browsers in which you tested your changes.

### CODE REVIEW

File: pages/jobs/[slug].tsx [hunk 1]
```diff
@@ -51,8 +51,8 @@ const JobPosting: NextPage<JobPageProps> = ({
     return (
         <Layout theme="LIGHT" preview={preview} jobsAvailable={jobsAvailable}>
             <Metadata
-                title={content.metadataTitle}
-                description={content.metadataDescription}
+                title={`${job.title} | Torchbox Careers`}
```
Comment (jhancock532) [inline pages/jobs/[slug].tsx:54]: I've gone for `| Torchbox Careers` as this is more specific, however across the site we've typically used just `| Torchbox` - should I update this to match the rest of the site?

Comment (Morsey187) [reply pages/jobs/[slug].tsx:54]: I'd update to " - Jobs | Torchbox" personally, I think this is more about consistency for google results, see below.
Screenshot 2022-11-03 at 09 01 43
Screenshot 2022-11-03 at 09 01 35

Comment (Morsey187) [reply pages/jobs/[slug].tsx:54]: Can we update all use of the "<Metadata" to, perhaps by appending this from the Metadata component instead?

**Review by Morsey187 — CHANGES_REQUESTED** [review]

File: pages/jobs/[slug].tsx (hunk 1 above)
Comment (Morsey187) [reply pages/jobs/[slug].tsx:54]: `Careers - jobs | Torchbox` might sound a little bit funny, so perhaps just the jobs page should be "- jobs | Torchbox".

I'd go for this change for now and bring it up with the SEO team for the whole site, not just the careers pages as currently there is no consitency.

Screenshot 2022-11-03 at 09 11 25

**Pull request merged**